Set the major .so version to 2 for all unixODBC shared libraries, to reflect
the incompatible ABI changes in 2.2.14.

diff -up unixODBC-2.3.3/Drivers/MiniSQL/Makefile.am.soname unixODBC-2.3.3/Drivers/MiniSQL/Makefile.am
--- unixODBC-2.3.3/Drivers/MiniSQL/Makefile.am.soname	2015-08-22 18:39:16.000000000 +0200
+++ unixODBC-2.3.3/Drivers/MiniSQL/Makefile.am	2015-08-25 12:27:47.005775376 +0200
@@ -193,6 +193,6 @@ EXTRA_DIST = \
 	_sqlFreeEnv.c \
 	_sqlFreeStmt.c 
 
-libodbcmini_la_LDFLAGS = -no-undefined -version-info 1:0:0 \
+libodbcmini_la_LDFLAGS = -no-undefined -version-info $(LIB_VERSION) \
 	-L@msql_libraries@ -lmsql -module
 
diff -up unixODBC-2.3.3/Drivers/nn/Makefile.am.soname unixODBC-2.3.3/Drivers/nn/Makefile.am
--- unixODBC-2.3.3/Drivers/nn/Makefile.am.soname	2014-05-05 10:27:13.000000000 +0200
+++ unixODBC-2.3.3/Drivers/nn/Makefile.am	2015-08-25 12:27:47.007775386 +0200
@@ -2,7 +2,7 @@ lib_LTLIBRARIES = libnn.la
 
 AM_CPPFLAGS = -I@top_srcdir@/include -I.
 
-libnn_la_LDFLAGS = -no-undefined  -version-info 1:0:0 -module
+libnn_la_LDFLAGS = -no-undefined  -version-info $(LIB_VERSION) -module
 
 EXTRA_DIST = \
 	nnconfig.h \
diff -up unixODBC-2.3.3/Drivers/Postgre7.1/Makefile.am.soname unixODBC-2.3.3/Drivers/Postgre7.1/Makefile.am
--- unixODBC-2.3.3/Drivers/Postgre7.1/Makefile.am.soname	2014-05-05 10:27:13.000000000 +0200
+++ unixODBC-2.3.3/Drivers/Postgre7.1/Makefile.am	2015-08-25 12:27:47.009775396 +0200
@@ -3,7 +3,7 @@ lib_LTLIBRARIES = libodbcpsql.la
 AM_CPPFLAGS = -I@top_srcdir@/include $(LTDLINCL)
 
 libodbcpsql_la_LDFLAGS = \
-	-version-info 2:0:0 \
+	-version-info $(LIB_VERSION) \
 	-no-undefined \
 	$(LIBSOCKET) $(LIBNSL) \
 	-export-dynamic \
diff -up unixODBC-2.3.3/Drivers/template/Makefile.am.soname unixODBC-2.3.3/Drivers/template/Makefile.am
--- unixODBC-2.3.3/Drivers/template/Makefile.am.soname	2014-05-05 10:27:13.000000000 +0200
+++ unixODBC-2.3.3/Drivers/template/Makefile.am	2015-08-25 12:27:47.012775410 +0200
@@ -2,7 +2,7 @@ lib_LTLIBRARIES = libtemplate.la
 
 AM_CPPFLAGS = -I@top_srcdir@/include -I. $(LTDLINCL)
 
-libtemplate_la_LDFLAGS = -no-undefined  -version-info 1:0:0 
+libtemplate_la_LDFLAGS = -no-undefined  -version-info $(LIB_VERSION) 
 
 EXTRA_DIST = \
 	driver.h \
diff -up unixODBC-2.3.3/DRVConfig/drvcfg1/Makefile.am.soname unixODBC-2.3.3/DRVConfig/drvcfg1/Makefile.am
--- unixODBC-2.3.3/DRVConfig/drvcfg1/Makefile.am.soname	2014-05-05 10:26:41.000000000 +0200
+++ unixODBC-2.3.3/DRVConfig/drvcfg1/Makefile.am	2015-08-25 12:27:47.014775420 +0200
@@ -2,7 +2,7 @@ lib_LTLIBRARIES = libodbcdrvcfg1S.la
 
 AM_CPPFLAGS = -I@top_srcdir@/include $(LTDLINCL)
 
-libodbcdrvcfg1S_la_LDFLAGS = -no-undefined  -version-info 1:0:0 -module
+libodbcdrvcfg1S_la_LDFLAGS = -no-undefined  -version-info $(LIB_VERSION) -module
 
 libodbcdrvcfg1S_la_SOURCES = drvcfg1.c
 	
diff -up unixODBC-2.3.3/DRVConfig/drvcfg2/Makefile.am.soname unixODBC-2.3.3/DRVConfig/drvcfg2/Makefile.am
--- unixODBC-2.3.3/DRVConfig/drvcfg2/Makefile.am.soname	2014-05-05 10:26:41.000000000 +0200
+++ unixODBC-2.3.3/DRVConfig/drvcfg2/Makefile.am	2015-08-25 12:27:47.016775430 +0200
@@ -2,7 +2,7 @@ lib_LTLIBRARIES = libodbcdrvcfg2S.la
 
 AM_CPPFLAGS = -I@top_srcdir@/include $(LTDLINCL)
 
-libodbcdrvcfg2S_la_LDFLAGS = -no-undefined  -version-info 1:0:0 -module
+libodbcdrvcfg2S_la_LDFLAGS = -no-undefined  -version-info $(LIB_VERSION) -module
 
 libodbcdrvcfg2S_la_SOURCES = drvcfg2.c
 	
diff -up unixODBC-2.3.3/DRVConfig/esoob/Makefile.am.soname unixODBC-2.3.3/DRVConfig/esoob/Makefile.am
--- unixODBC-2.3.3/DRVConfig/esoob/Makefile.am.soname	2014-05-05 10:26:41.000000000 +0200
+++ unixODBC-2.3.3/DRVConfig/esoob/Makefile.am	2015-08-25 12:27:47.017775434 +0200
@@ -2,7 +2,7 @@ lib_LTLIBRARIES = libesoobS.la
 
 AM_CPPFLAGS = -I@top_srcdir@/include $(LTDLINCL)
 
-libesoobS_la_LDFLAGS = -no-undefined  -version-info 1:0:0 -module
+libesoobS_la_LDFLAGS = -no-undefined  -version-info $(LIB_VERSION) -module
 
 libesoobS_la_SOURCES = esoobS.c
 	
diff -up unixODBC-2.3.3/DRVConfig/Mimer/Makefile.am.soname unixODBC-2.3.3/DRVConfig/Mimer/Makefile.am
--- unixODBC-2.3.3/DRVConfig/Mimer/Makefile.am.soname	2014-05-05 10:26:41.000000000 +0200
+++ unixODBC-2.3.3/DRVConfig/Mimer/Makefile.am	2015-08-25 12:27:47.019775444 +0200
@@ -2,7 +2,7 @@ lib_LTLIBRARIES = libmimerS.la
 
 AM_CPPFLAGS = -I@top_srcdir@/include $(LTDLINCL)
 
-libmimerS_la_LDFLAGS = -no-undefined -version-info 1:0:0 -module
+libmimerS_la_LDFLAGS = -no-undefined -version-info $(LIB_VERSION) -module
 
 libmimerS_la_SOURCES = mimerS.c
 	
diff -up unixODBC-2.3.3/DRVConfig/MiniSQL/Makefile.am.soname unixODBC-2.3.3/DRVConfig/MiniSQL/Makefile.am
--- unixODBC-2.3.3/DRVConfig/MiniSQL/Makefile.am.soname	2014-05-05 10:26:41.000000000 +0200
+++ unixODBC-2.3.3/DRVConfig/MiniSQL/Makefile.am	2015-08-25 12:27:47.020775449 +0200
@@ -2,7 +2,7 @@ lib_LTLIBRARIES = libodbcminiS.la
 
 AM_CPPFLAGS = -I@top_srcdir@/include $(LTDLINCL)
 
-libodbcminiS_la_LDFLAGS = -no-undefined  -version-info 1:0:0 -module
+libodbcminiS_la_LDFLAGS = -no-undefined  -version-info $(LIB_VERSION) -module
 
 libodbcminiS_la_SOURCES = odbcminiS.c
 	
diff -up unixODBC-2.3.3/DRVConfig/MySQL/Makefile.am.soname unixODBC-2.3.3/DRVConfig/MySQL/Makefile.am
--- unixODBC-2.3.3/DRVConfig/MySQL/Makefile.am.soname	2014-05-05 10:26:41.000000000 +0200
+++ unixODBC-2.3.3/DRVConfig/MySQL/Makefile.am	2015-08-25 12:27:47.023775464 +0200
@@ -2,7 +2,7 @@ lib_LTLIBRARIES = libodbcmyS.la
 
 AM_CPPFLAGS = -I@top_srcdir@/include $(LTDLINCL)
 
-libodbcmyS_la_LDFLAGS = -no-undefined  -version-info 1:0:0 -module
+libodbcmyS_la_LDFLAGS = -no-undefined  -version-info $(LIB_VERSION) -module
 
 libodbcmyS_la_SOURCES = odbcmyS.c
 	
diff -up unixODBC-2.3.3/DRVConfig/nn/Makefile.am.soname unixODBC-2.3.3/DRVConfig/nn/Makefile.am
--- unixODBC-2.3.3/DRVConfig/nn/Makefile.am.soname	2014-05-05 10:26:41.000000000 +0200
+++ unixODBC-2.3.3/DRVConfig/nn/Makefile.am	2015-08-25 12:27:47.024775469 +0200
@@ -2,7 +2,7 @@ lib_LTLIBRARIES = libodbcnnS.la
 
 AM_CPPFLAGS = -I@top_srcdir@/include $(LTDLINCL)
 
-libodbcnnS_la_LDFLAGS = -no-undefined  -version-info 1:0:0 -module
+libodbcnnS_la_LDFLAGS = -no-undefined  -version-info $(LIB_VERSION) -module
 
 libodbcnnS_la_SOURCES = drvcfg.c
 	
diff -up unixODBC-2.3.3/DRVConfig/oplodbc/Makefile.am.soname unixODBC-2.3.3/DRVConfig/oplodbc/Makefile.am
--- unixODBC-2.3.3/DRVConfig/oplodbc/Makefile.am.soname	2014-05-05 10:26:41.000000000 +0200
+++ unixODBC-2.3.3/DRVConfig/oplodbc/Makefile.am	2015-08-25 12:27:47.026775478 +0200
@@ -2,7 +2,7 @@ lib_LTLIBRARIES = liboplodbcS.la
 
 AM_CPPFLAGS = -I@top_srcdir@/include $(LTDLINCL)
 
-liboplodbcS_la_LDFLAGS = -no-undefined  -version-info 1:0:0 -module
+liboplodbcS_la_LDFLAGS = -no-undefined  -version-info $(LIB_VERSION) -module
 
 liboplodbcS_la_SOURCES = oplodbc.c
 	
diff -up unixODBC-2.3.3/DRVConfig/Oracle/Makefile.am.soname unixODBC-2.3.3/DRVConfig/Oracle/Makefile.am
--- unixODBC-2.3.3/DRVConfig/Oracle/Makefile.am.soname	2014-05-05 10:26:41.000000000 +0200
+++ unixODBC-2.3.3/DRVConfig/Oracle/Makefile.am	2015-08-25 12:27:47.027775483 +0200
@@ -2,7 +2,7 @@ lib_LTLIBRARIES = liboraodbcS.la
 
 AM_CPPFLAGS = -I@top_srcdir@/include $(LTDLINCL)
 
-liboraodbcS_la_LDFLAGS = -no-undefined  -version-info 1:0:0 -module
+liboraodbcS_la_LDFLAGS = -no-undefined  -version-info $(LIB_VERSION) -module
 
 liboraodbcS_la_SOURCES = oraodbcS.c
 	
diff -up unixODBC-2.3.3/DRVConfig/PostgreSQL/Makefile.am.soname unixODBC-2.3.3/DRVConfig/PostgreSQL/Makefile.am
--- unixODBC-2.3.3/DRVConfig/PostgreSQL/Makefile.am.soname	2014-05-05 10:26:41.000000000 +0200
+++ unixODBC-2.3.3/DRVConfig/PostgreSQL/Makefile.am	2015-08-25 12:27:47.029775493 +0200
@@ -2,7 +2,7 @@ lib_LTLIBRARIES = libodbcpsqlS.la
 
 AM_CPPFLAGS = -I@top_srcdir@/include $(LTDLINCL)
 
-libodbcpsqlS_la_LDFLAGS = -no-undefined  -version-info 1:0:0 -module
+libodbcpsqlS_la_LDFLAGS = -no-undefined  -version-info $(LIB_VERSION) -module
 
 libodbcpsqlS_la_SOURCES = odbcpsqlS.c
 	
diff -up unixODBC-2.3.3/DRVConfig/sapdb/Makefile.am.soname unixODBC-2.3.3/DRVConfig/sapdb/Makefile.am
--- unixODBC-2.3.3/DRVConfig/sapdb/Makefile.am.soname	2014-05-05 10:26:41.000000000 +0200
+++ unixODBC-2.3.3/DRVConfig/sapdb/Makefile.am	2015-08-25 12:27:47.031775503 +0200
@@ -2,7 +2,7 @@ lib_LTLIBRARIES = libsapdbS.la
 
 AM_CPPFLAGS = -I@top_srcdir@/include $(LTDLINCL)
 
-libsapdbS_la_LDFLAGS = -no-undefined  -version-info 1:0:0 -module
+libsapdbS_la_LDFLAGS = -no-undefined  -version-info $(LIB_VERSION) -module
 
 libsapdbS_la_SOURCES = sapdb.c
 
diff -up unixODBC-2.3.3/DRVConfig/tds/Makefile.am.soname unixODBC-2.3.3/DRVConfig/tds/Makefile.am
--- unixODBC-2.3.3/DRVConfig/tds/Makefile.am.soname	2014-05-05 10:26:41.000000000 +0200
+++ unixODBC-2.3.3/DRVConfig/tds/Makefile.am	2015-08-25 12:27:47.032775507 +0200
@@ -2,7 +2,7 @@ lib_LTLIBRARIES = libtdsS.la
 
 AM_CPPFLAGS = -I@top_srcdir@/include $(LTDLINCL)
 
-libtdsS_la_LDFLAGS = -no-undefined  -version-info 1:0:0 -module
+libtdsS_la_LDFLAGS = -no-undefined  -version-info $(LIB_VERSION) -module
 
 libtdsS_la_SOURCES = tdsS.c
 	
diff -up unixODBC-2.3.3/DRVConfig/txt/Makefile.am.soname unixODBC-2.3.3/DRVConfig/txt/Makefile.am
--- unixODBC-2.3.3/DRVConfig/txt/Makefile.am.soname	2014-05-05 10:26:41.000000000 +0200
+++ unixODBC-2.3.3/DRVConfig/txt/Makefile.am	2015-08-25 12:27:47.034775517 +0200
@@ -2,7 +2,7 @@ lib_LTLIBRARIES = libodbctxtS.la
 
 AM_CPPFLAGS = -I@top_srcdir@/include $(LTDLINCL)
 
-libodbctxtS_la_LDFLAGS = -no-undefined  -version-info 1:0:0 -module
+libodbctxtS_la_LDFLAGS = -no-undefined  -version-info $(LIB_VERSION) -module
 
 libodbctxtS_la_SOURCES = drvcfg.c
 	
